shell.php

```diff
@@ -427,6 +427,13 @@ function getQueryString() {
                 xhr.send(getQueryString());
             }
 
+            function focusShellCmd() {
+                var selection = window.getSelection();
+                if(!selection.toString()) {
+                    document.getElementById('shell-cmd').focus();
```

Comment:
No need to call `getElementById()` here, there is already a `eShellCmdInput` variable that contains this element

Reply:
No. The changes which you mention not correct. The code in this PR will execute every click, not only onload.